### ISSUE TITLE
"CodePush - Release" task missing after instalation to Azure DevOps (VSTS)

### ISSUE THREAD

**Issue by Imroath** [Microsoft/code-push-vsts-extension#30]
I have installed the ms-vsclient.code-push-1.3.1.vsix extension in our AzureDevOps but we are missing “CodePush – Release” task. I can see 3 other tasks (CodePush - Patch, CodePush - Release(Cordova) and CodePush - Promote). What we want to do is to release React-Native app, and without the CodePush - Release task, we can't do it.

**Comment by alexandergoncharov** [comment]
Hi @Imroath ,
Sorry for this long delay.

Yeah, we don't have `release-react` command. For releasing react app you should use `react-native prepare` and then `CodePush-release`. For more info please check this docs: https://marketplace.visualstudio.com/items?itemName=ms-vsclient.react-native-extension

Also, I see that you missed `CodePush-release` task. It is strange. Could you please clarify what approach have you used for installing CodePush extension? I would like to to your step and reproduce this issue.

Thanks,
Alexander

**Comment by Imroath** [comment]
> Hi @Imroath ,
> Also, I see that you missed `CodePush-release` task. It is strange. Could you please clarify what approach have you used for installing CodePush extension? I would like to to your step and reproduce this issue.

Yes, this issue is strange. But all I did was to navigate to marketplace, https://marketplace.visualstudio.com/items?itemName=ms-vsclient.code-push and click "Get it free". Then I installed extension in my Azure DevOps Organization. But the CodePush-Release task was missing after instalation. My work around was to download and extract the package and then use tfx-cli to install CodePush-Release task manually just as we do with our custom tasks. And it worked fine.

**Comment by Imroath** [comment]
Maybe the source of an issue is that you are using a depreciated and outdated vsts-task-lib? 
Maybe you should use azure-pipelines-task-lib instead?
